src/ripple/ledger/impl/ApplyStateTable.cpp

```diff
@@ -562,10 +562,10 @@ ApplyStateTable::getForMod (ReadView const& base,
             auto const& item = iter->second;
             if (item.first == Action::erase)
             {
-                // VFALCO We need to think about throwing
-                //        an exception or calling LogicError
-                JLOG(j.fatal()) <<
-                    "Trying to thread to deleted node";
+                // The Destination of an Escrow or a PayChannel may have been
```

Comment:
Dst of escrow cannot be deleted

Reply:
Follow-up on my own comment. I'm wrong about the above. I forgot about pre-fix escrows. So no change needed.